workflows/cloudify_system_workflows/deployment_environment.py

```diff
@@ -39,10 +56,15 @@ def generate_create_dep_tasks_graph(ctx,
             ctx.execute_task('cloudify_agent.operations.install_plugins',
                              kwargs={'plugins': plugins_to_install}))
 
-    sequence.add(
-        ctx.send_event('Starting deployment policy engine core'),
-        ctx.execute_task('riemann_controller.tasks.create',
-                         kwargs=policy_configuration or {}))
+    if _should_create_policy_engine_core(policy_configuration):
+        sequence.add(
+            ctx.send_event('Starting deployment policy engine core'),
+            ctx.execute_task('riemann_controller.tasks.create',
+                             kwargs=policy_configuration))
+    else:
+        sequence.add(
```

Comment:
maybe update the delete deployment part so that is says something like
`Stopping deployment policy engine core (if applicable)`
or some other wording